.github/templates/playwright/action.yml

```
@@ -17,18 +14,28 @@ runs:
     - name: 📦 Install Playwright system dependencies
       shell: bash
       run: pnpm exec playwright install-deps
-      working-directory: ${{ inputs.working-directory }}
+
+    - name: 🗑️ Delete existing screenshots
+      if: ${{ inputs.update-snapshots == 'true' }}
+      shell: bash
+      run: |
+        rm -rf  ./packages/chartjs-plugin/playwright/snapshots
+        rm -rf ./packages/headless/playwright/snapshots
+        rm -rf ./packages/sit-onyx/playwright/snapshots
```

Would something like this work?

```suggestion
        rm -rf  **/playwright/snapshots
```